Doc/library/winreg.rst

```diff
@@ -159,8 +161,12 @@ This module offers the following functions:
 
    *reserved* is a reserved integer, and must be zero. The default is zero.
 
-   *access* is an integer that specifies an access mask that describes the desired
-   security access for the key.  Default is :const:`KEY_WOW64_64KEY`.  See
+   *access* is an integer that specifies an access mask that describes the
+   desired security access for the key.  Default is :const:`KEY_WOW64_64KEY`.
+   On 32-bit Windows, the access constants :const:`KEY_WOW64_64KEY` and
+   :const:`KEY_WOW64_32KEY` are ignored, thus 0 should be passed in
+   for this parameter as a workaround.
+   See
    :ref:`Access Rights <access-rights>` for other allowed values.
```

Comment:
```suggestion
   On 32-bit Windows, the WOW64 constants are ignored.
   See :ref:`Access Rights <access-rights>` for other allowed values.
```

I think we can simplify down to this. There's no need to override the default value, as the flags are ignored.